bootstrap_cfn/elb.py

```diff
@@ -84,3 +84,31 @@ def set_ssl_certificates(self, ssl_config, stack_name):
 
         return updated_load_balancers
 
+    def list_domain_names(self, stack_name):
+        """
+        Return a list of dicts, each containing the ELB name and corresponding DNS Name for
+        each ELB in a given environment.
+
+        Args:
+            stack name
+
+        Returns:
+            list of dict: [{'elb_name': string, 'dns_name': string}]
+
+        Raises:
+            The boto call raises self.ResponseError(response.status, response.reason, body)
+            if the AWS call returns an error, for example if the ELB name does not exist
+        """
+        lb_name_dns = []
+        load_balancer_resources = self.cfn.get_stack_load_balancers(stack_name)
+        for l in load_balancer_resources:
```

Comment:
Can this for loop be simplified to something like:

```python
lb_ids = [l.physical_resource_id for l in load_balancer_resources]
lb_details = self.conn_elb.get_all_load_balancers(load_balancer_names=lb_ids)
```

We can then check that the number of results we get back is the same as the number of ids we passed.